examples/tensorflow-mnist-pixel-threshold/demo.ipynb

```diff
@@ -224,44 +248,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "response_custom_plugins = restapi_client.get_custom_task_plugin(name=\"evaluation\")\n",
-    "#restapi_client.delete_custom_task_plugin(name=\"evaluation\")\n",
-    "if response_custom_plugins is None or \"Not Found\" in response_custom_plugins.get(\"message\", []):\n",
-    "    response_custom_plugins = restapi_client.upload_custom_plugin_package(\n",
-    "        custom_plugin_name=\"evaluation\",\n",
-    "        custom_plugin_file=CUSTOM_PLUGINS_EVALUATION_TAR_GZ,\n",
-    "    )\n",
-    "\n",
-    "response_custom_plugins\n",
-    "\n",
-    "response_custom_plugins = restapi_client.get_custom_task_plugin(name=\"pixel_threshold\")\n",
-    "#restapi_client.delete_custom_task_plugin(name=\"pixel_threshold\")\n",
-    "if response_custom_plugins is None or \"Not Found\" in response_custom_plugins.get(\"message\", []):\n",
-    "    response_custom_plugins = restapi_client.upload_custom_plugin_package(\n",
-    "        custom_plugin_name=\"pixel_threshold\",\n",
-    "        custom_plugin_file=CUSTOM_PLUGINS_PIXEL_THRESHOLD_TAR_GZ,\n",
-    "    )\n",
-    "\n",
-    "response_custom_plugins"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "If at any point you need to update one or more files within the `evaluation` and `pixel_threshold` plugin packages, you will need to unregister/delete the custom task plugin first using the REST API.\n",
-    "This can be done as follows,"
+    "restapi_client.list_queues()"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "```python\n",
-    "# Delete the 'evaluation' custom task plugin package\n",
-    "restapi_client.delete_custom_task_plugin(name=\"evaluation\")\n",
-    "restapi_client.delete_custom_task_plugin(name=\"pixel_threshold\")\n",
-    "```"
+    "restapi_client.delete_custom_task_plugin(name = \"pixel_threshold\")"
```

Comment:
It feels a bit weird to have this in here and not also have a call to reupload the plugins. It does get in the way of clicking through the notebook as you have to be careful to reupload the plugins manually before moving on to the next step. Maybe this could be moved to a text block if we don't want to include a reupload call?

Reply:
Thanks for catching this, I'd left that in when editing the plugins and testing.